.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -0,0 +1,78 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔.
+
+---
+
+<!---
+Before you publish a bug report, please make sure you have have made a reasonable effort to eliminate any outside causes.
+
+* Check the official documentation for the resource that errors to see whether you have configured it incorrectly, or it's being used incorrectly within the module.
+* Make sure you have the latest stable version of your cloud provider version and see if anything has change (which is related to the resource that errors) in newer releases of the provider.
+* Made sure you have performed validation (`terraform fmt`, `terraform validate` or `tflint`) against your terraform workspace.
+* Searched for any changes or issues regarding the terraform provider in question (eg. https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md)
+
+If you are running into one of these scenarios, we recommend you reaching out to the terraform community, see information further down.
```

Comment:
Are we keeping that bar too high by this? The last point of looking though the terraform provider repo for issues might be overkill. It's good if they do but I'd rather have people creating bug reports then skipping it because its too much work.

Reply:
Ok, I'll remove that one